src/main/kotlin/de/bund/digitalservice/useid/events/EventController.kt

```diff
@@ -36,9 +36,9 @@ class EventController(eventService: EventService) {
     }
 
     /**
-     * These endpoints receive events from the eID client (i.e. Ident-App) and publish them to the respective consumer.
+     * Receive events from the eID client (i.e. Ident-App) and publish them to the respective consumer.
      */
-    fun publishEvent(event: ServerSentEvent<Any>, widgetSessionId: UUID): Mono<ResponseEntity<Nothing?>> {
+    fun publishEvent(event: ServerSentEvent<Any>, widgetSessionId: UUID): Mono<ResponseEntity<Nothing>> {
```

Comment:
<img width="959" alt="image" src="https://user-images.githubusercontent.com/7918047/209071494-b9fe1639-a537-4880-b97a-3c89b89eebc6.png">
Is this not like that for you? Any idea? 🤔

Reply:
What? No, never seen this. Let's have a look together.